### DISCUSSION TITLE
Support for wildcard subdomains

### DISCUSSION THREAD

**Discussion by awwong1** [plausible/analytics#2404] [Anything Else]
> If you would like to track a subdomain such as blog.yourdomain.com then please include the subdomain. Plausible Analytics essentially sees different subdomains as different sites.
https://plausible.io/docs/add-website#do-you-want-to-track-a-subdomain

Also related: https://plausible.io/docs/plausible-script#is-there-a-roll-up-view

I have a Cloudflare Pages application hosted on `u0.vc` and `www.u0.vc`.
Cloudflare Pages deployments also exist at `*.u0-vc.pages.dev` (e.g. https://4ba26d3e.u0-vc.pages.dev/)

I would like to have a script tag that will track analytics for each of these domains and aggregate them into a single dashboard.

**Comment by metmarkosaric** [comment]
hi @awwong1! you could use the same snippet on all these if the only thing you want is one dashboard for them all. see https://plausible.io/docs/plausible-script#can-i-use-the-same-snippet-across-my-domain-and-subdomain

**Comment by awwong1** [comment]
Just so I understand, I can do something like
```html
<script defer data-domain="u0.vc" data-api="/opt/event" src="/opt/script.outbound-links.js"></script>
```
in all of my pages and all of these stats will aggregate in https://plausible.io/u0.vc ?

relevant: https://github.com/udiaca/u0.vc/blob/464645ec465b5889e6c8f3b8e3c3ed24d26651e9/src/layouts/BaseLayout.astro#L26

**Reply by metmarkosaric** [reply]
yes, should work fine like that. you can try the different approaches we offer and pick what fits best for your needs
